src/main/app/package.json

```diff
@@ -54,6 +55,7 @@
     "@angular/language-service": "^17.2.1",
     "@types/jest": "^29.5.12",
     "@types/node": "^20.12.2",
+    "@types/uuid": "^9.0.8",
```

Comment:
The API is, but they're generally not generated on the client side. Even this package is something that is already referenced indirectly as a transitive dependency, but I felt it would be best to also add it as a direct dependency now we're using it directly.

Reply:
clear!